--- /usr/share/X11/xkb/rules/evdev.org	2017-06-24 02:56:32.407734316 +0900
+++ /usr/share/X11/xkb/rules/evdev	2017-06-24 03:12:20.320886265 +0900
@@ -1119,3 +1119,14 @@
   numpad:mac			=	+numpad(mac)
   numpad:microsoft		=	+numpad(microsoft)
   numpad:shift3       		=	+numpad(shift3)
+
+! option = symbols
+  jphenk:rshift  = +jphenk(rshift)
+  jphenk:rctrl   = +jphenk(rctrl)
+  jphenk:escape  = +jphenk(escape)
+  jphenk:ralt    = +jphenk(ralt)
+  jpmuhen:lshift = +jpmuhen(lshift)
+  jpmuhen:lctrl  = +jpmuhen(lctrl)
+  jpmuhen:escape = +jpmuhen(escape)
+  jpmuhen:lalt   = +jpmuhen(lalt)
+
